src/conmon.c

```diff
@@ -572,7 +572,10 @@ static gboolean oom_cb_cgroup_v2(int fd, GIOCondition condition, G_GNUC_UNUSED g
 	gboolean ret = G_SOURCE_REMOVE;
 
 	/* Drop the inotify events.  */
-	read(fd, &events, sizeof(events));
+	ssize_t num_read = read(fd, &events, sizeof(events));
+	if (num_read < 0) {
+		nwarn("Failed to read events");
```

Comment:
Can we use `pwarn` instead? `pwarn` is using strerror(3) to translate errno into something human readable, which could come in handy for debugging.

Reply:
Yes sure, I adapted it as suggested.